src/settingsdialog.cpp

```diff
@@ -290,3 +297,128 @@ void SettingsDialog::addCallgraphPage()
                                      callgraphPage->functionColor->color().name());
     });
 }
+
+void SettingsDialog::addSshPage()
+{
+    const auto sshPath = QStandardPaths::findExecutable(QLatin1String("ssh"));
```

Comment:
QStringLiteral

Reply:
`const auto sshPath = QStandardPaths::findExecutable(QStringLiteral("ssh"));` or `const auto sshPath = QStringLiteral("ssh"));`

Reply:
the former